pkg/crds/controllers/batch/batchjob_controller_helpers.go

```diff
@@ -301,6 +301,7 @@ func (r *BatchJobReconciler) desiredWorkerJob(batchJob batch.BatchJob, apiSpec s
 				"handlerID":      apiSpec.HandlerID,
 				"jobID":          batchJob.Name,
 				"cortex.dev/api": "true",
+				"worker":         "true",
```

Comment:
nit: we could do like async: "cortex.dev/batch": "worker" (for worker) and  "cortex.dev/batch": "enqueuer" for enqueuer